src/pages/potd.tsx

```
@@ -1,407 +1,423 @@
-                      <li key={s.uid} className={styles.solverItem}>
-                        <div className={styles.solverInfo}>
-                          <span className={styles.rank}>#{i + 1}</span>
-                          <span className={styles.username}>{s.username || "User"}</span>
-                        </div>
-                        <span className={styles.solveCount}>
-                          {s.isFallback ? "-" : `${s.weeklyPotdSolves} ${s.weeklyPotdSolves === 1 ? "Day" : "Days"}`}
-                        </span>
+
+                // Changed the Name of the Leaderboard Weekly Top to Leaderboard Top 🎉
+
+                <div className={styles.sidebarCard}>
```

ESLint

[error] Comments inside children section of tag should be placed inside braces

(react/jsx-no-comment-textnodes)